vite.plugins.ts

```diff
@@ -151,13 +153,15 @@ export const replicaForwardPlugin = ({
 
       // Try to read the canister ID of a potential canister called <subdomain>
       // and if found forward to that
-      try {
-        const canisterId = execSync(`dfx canister id ${subdomain}`)
-          .toString()
-          .trim();
-        console.log("Forwarding to", canisterId);
-        return forwardToReplica({ canisterId });
-      } catch {}
+      if (domain === "localhost") {
```

Comment:
I found this logic confusing to follow. I.e. it was not immediately clear to me that when accessing `localhost:5173` the `domain` is the empty string and thus this `if` is not triggered (and the _subdomain_ is `localhost`).

Maybe add a comment for other lost souls like me. 🙃

Reply:
Much clearer, thanks! 👍